config/bundles.php

```diff
@@ -7,16 +7,13 @@
     Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
     Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
     Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
-    Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle::class => ['all' => true],
```

Comment:
This bundle is dead, see https://github.com/doctrine/DoctrineCacheBundle/releases/tag/1.4.0